test/rubygems/test_gem_dependency_installer.rb

```diff
@@ -312,7 +313,7 @@ def test_install_doesnt_upgrade_installed_dependencies
     end
 
     assert_equal %w[a-2 b-1], Gem::Specification.map(&:full_name)
-    assert_equal %w[b-1], inst.installed_gems.map { |s| s.full_name }
+    assert_equal %w[b-1], inst.installed_gems.map(&:full_name)
```

Comment:
Please don't mix stylistic change to feature/bugfix pull request.

Reply:
Sorry, I wouldn't have changed it except for the fact that the right above it uses `(&:full_name)`

I assume you would you like it changed (removed)?

Reply:
Yes. Remove/Revert it.